src/call.h

```diff
@@ -76,6 +78,8 @@ struct CallData {
     }
 };
 
+extern std::vector<CallData *> calls_assembled;
```

Comment:
Does `calls_assembled` need to be exposed to all people who include `call.h`? (Fine if it does, just checking)

Reply:
Yes, I need it to capture offset buffers. Should I document that?

Reply:
No I think that's alright, I was just checking :+1: